modules/ossm-cr-example.adoc

```diff
@@ -211,10 +211,6 @@ spec:
         config:
           env: {}
           envSecrets: {}
-        persistence:
-          storageClassName: ""
-          accessMode: ReadWriteOnce
-          capacity: 5Gi
         service:
```

Comment:
I think these are okay to leave, although we might want to have a valid value for `storageClassName`.  We should also add an `enabled: false` entry under persistence too.  I think there's an open issue where `capacity` does not get applied.

Reply:
@xyun This CR example is for 2.0
@rcernich   Do you have an example that includes the changes you mention?  Or do you want to open a OSSMDOC issue and get those updates in a different PR?  (That is, are we OK to merge these changes or should I make those changes now?)